pandas/tests/internals/test_internals.py

```diff
@@ -925,13 +924,13 @@ def assert_take_ok(mgr, axis, indexer):
         for mgr in self.MANAGERS:
             for ax in range(mgr.ndim):
                 # take/fancy indexer
-                assert_take_ok(mgr, ax, [])
-                assert_take_ok(mgr, ax, [0, 0, 0])
-                assert_take_ok(mgr, ax, lrange(mgr.shape[ax]))
+                assert_take_ok(mgr, ax, indexer=[])
```

Comment:
What was the reason for adding the keyword here?

Reply:
That was just to aid readability, I found the meaning of those lists became easier to understand. Though I see now it hasnt been done in all the function calls.